### PR TITLE
Capture kernel boot messages in journal.

### LINKED ISSUE #1113: Create doc on how to capture kernel boot messages in the journal
**Do you think Clear Linux documentation needs a new document? Please describe.**
Original issue: https://community.clearlinux.org/t/systemd-journal-missing-kernel-boot-messages/2095/2

Auke implemented ability to capture kernel boot messages on systemd-journald ver 245.  This short doc. shows how to enable this feature.

**Should the new document be a guide, a reference, or a tutorial?**
Put in under the Guides/Kernel

### PR DESCRIPTION
**Pull request by bktan8**
Closes #1113

Signed-off-by: Bun K Tan <bun.k.tan@intel.com>

### CODE REVIEW

File: source/guides/kernel/kernel-boot-msg.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.. _kernel-boot-msg:
+
+Capture Kernel Boot Messages in the Journal
+###########################################
+
+On |CL|, by default, kernel boot messages are not captured in the journal 
+and reported as "Missed" messages.  This is done in order to provide fast boot
```
Comment (mvincerx) [inline source/guides/kernel/kernel-boot-msg.rst:7]: `This` pronoun gets lost in the shuffle. Consider something like: 

By default |CL| does not capture kernel boot messages in the journal logs, where they're reported as "Missed" messages.  This design decision for the  [boot-loader?] was made to provide a fast boot time. On the other hand, if you wish to see the message, follow this guide.

Something along those lines. Does that make sense?

Comment (bktan8) [reply source/guides/kernel/kernel-boot-msg.rst]: made some changes per your suggestion.

File: source/guides/kernel/kernel-boot-msg.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+   :linenos:
+   :emphasize-lines: 4
+
+   -- Reboot --
+   Apr 10 19:55:43 kernel systemd-journald[300]: Journal started
+   Apr 10 19:55:43 kernel systemd-journald[300]: Runtime Journal (/run/log/journal/d01862ca79d1064ea379cd715cfdd53a) is 5.8M, max 47.0M, 41.1M free.
+   Apr 10 19:55:43 kernel systemd-journald[300]: Missed 2233 kernel messages
+   Apr 10 19:55:43 kernel systemd[1]: Started Journal Service.
+
+This design decision was made to provide a faster boot performance.
```
Comment (mvincerx) [inline source/guides/kernel/kernel-boot-msg.rst:21]: So, we also need to respect to the G o o g l e monster too. Need to feed it what it wants--in the hopes that this gets indexed. That means we need to state the purpose of the doc. Right away. Move Ln 22 to follow sentence on Ln 7.  We can live without Ln 21; it's optional to include it in the intro.  And feel free to re-word current Ln 22.

Comment (bktan8) [reply source/guides/kernel/kernel-boot-msg.rst:21]: ok. done.

Comment (mvincerx) [reply source/guides/kernel/kernel-boot-msg.rst:21]: Hmm. Nothing changed. Maybe check how you squashed commits.

File: source/guides/kernel/kernel-boot-msg.rst (hunk 1 above)
Comment (bktan8) [reply source/guides/kernel/kernel-boot-msg.rst]: my bad.  i forgot to "add."  

for some reason, the automated validation process is taking much longer than usual.

File: source/guides/kernel/kernel-boot-msg.rst (hunk 2 above)
Comment (bktan8) [reply source/guides/kernel/kernel-boot-msg.rst]: my bad.  i forgot to "add."  

for some reason, the automated validation process is taking much longer than usual.

Comment (mvincerx) [reply source/guides/kernel/kernel-boot-msg.rst]: No worries. Not sure why automated validation seems to take longer.

**Comment by mvincerx** [comment]
LGTM.

**Pull request merged**